### ISSUE TITLE
pkg installs command.com and kernel.sys in the root C: directory

### ISSUE THREAD

**Issue by mateuszviste** [SvarDOS/bugz#100]
kernel.sys and command.com files are installed into C:\ by pkg. This makes it impossible to install SvarDOS on an A: drive or D:.
Installing on A: is a little bit eccentric, but might be a very valid scenario. Installing on D: is even more valid since the EDR kernel will name its default (boot) drive D: if it is the second FAT partition in the MBR.

Instead of putting kernel.sys and command.com into C:\, pkg should probably ask DOS for the "boot drive" (int 21/AX=3305h) and copy kernel.sys/command.com to the root directory of that.

**Comment by mateuszviste** [comment]
[implemented](http://svn.svardos.org/comp.php?repname=SvarDOS&compare[]=%2Fpkg@1800&compare[]=%2Fpkg@1878)

**Comment by mateuszviste** [comment]
This smart implementation sucks. I bricked my 386 today due to a little kernel glitch, so I booted from a floppy and wanted to reinstall another kernel to rescue the system. And of course pkg insisted that the kernel.sys file should land on A: since this is the boot drive... and I had no way to tell it otherwise.

It would be better to have the "system drive" configurable through pkg.cfg instead of autedetecting it.
